litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -104,10 +104,14 @@ rules:
     resources:
       - pods
       - pods/exec
+      - services
```

Comment:
do we need to add services as a resource to namespace manifest too? @rajdas98

Reply:
https://github.com/litmuschaos/litmus/pull/2127#discussion_r501459039